nitrocli/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 Unreleased
 ----------
+- Use the nitrokey crate for the `open` and `close` commands instead of
+  directly communication with the Nitrokey device.
```

Comment:
*`communicating`

I've kept all bullet points without an explicit full stop at the end (although they may be a full sentence). Mind doing the same for the sake of consistency?

Reply:
I try to; sometimes I just don’t notice things that I am so used to.